lt/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="lt">
+<head>
+  <title>Ne klauskite, kad klausti, klauskite iš karto</title>
+  <meta property="og:title" content="Ne klauskite, kad klausti, klauskite iš karto" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com/ru/" />
+  <meta property="og:locale" content="lt_LT" />
+  <meta property="og:image" content="https://dontasktoask.com/favicon.png"/>
+  <link rel="icon" href="/favicon.png" type="image/png" />
+  <link rel="stylesheet" type="text/css" href="/style.css">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Open+Sans">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+</head>
+<body>
+  <main>
+    <h1>Ne klauskite, kad klausti, klauskite iš karto</h1>
+    <p>
+	  Pokalbiuose, kuriuose aš būnu, kartais pasitaiko, kad kažkas klausia
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Kažkas iš čia rašo ant Java kalbos?
+      </p>
+    </blockquote>
+    <p>
+      Tai yra blogas klausimų formatas dėl daugelių priežasčių. <em>Tiesą sakant</em>
+      Žmogus klausia:
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+		Kažkas čia rašo "Java" kalba ir nori skirti laiko mano analizei problemos,
+		kad ir kuo tai baigtųsi,
+		net jei tai yra tikrai nėra labai susijęs su Java ir bet kas,
+		neturintis žinių apie Java ar galėtumėte atsakyti į mano klausimą?
+      </p>
+    </blockquote>
+    <p>
+		Yra daug priežasčių,kodėl žmonės, turintys žinių tinkamoje srityje,
+		gali neatsakyti.Tokiu būdu užduodami klausimą,jūs klausiate apie daugiau,nei
+		nei atrodo iš pirmo žvilgsnio.
+    </p>
+    <p>
+	Jūs klausiate žmonių, ar jie nori prisiimti atsakomybę. Tu abejoja jų pasitikėjimu savo įgūdžiais.
+	Taip pat žr.: nepagrįstai atsisakyti kitų žmonių pagalbos. Pavyzdžiui, aš dažnai atsakyti į klausimus,
+	susijusius su kalbomis ar bibliotekomis, kurias aš niekada nenaudojamas,
+	nes atsakymai yra tik sveikas protas (programuotojo požiūriu).
+    </p>
+    <p>	
+	Alternatyviai šį klausimą galima perskaityti kaip..
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+		Turiu klausimą, susijusį su "Java", bet labai tingiu, kad jį suformuluočiau nebent čia yra kas nors,
+		kas galėtų į tai atsakyti.
+      </p>
+    </blockquote>
+    <p>
+		.. kuris tiesiog kalba apie jūsų tingumą. Jei nenorite atlikti darbo, Kodėl, norėdami išspręsti jūsų problemą,
+		turėtume to norėti?

```

Comment:
this sentence also looks very weird and odd, i cant even make sense out of it rlly lol

instead, u should say smt like : 

> ... o tai tiesiog tingu. Jei nenorite atlikti darbo, kad išspręstumėte savo problemą, kodėl turėtume tai padaryti mes?